test/integration/krane_test.rb

```diff
@@ -53,8 +53,8 @@ def test_render_black_box
   end
 
   def test_deploy_black_box_success
-    setup_template_dir("hello-cloud") do |target_dir|
-      flags = "-f #{target_dir} --render-erb --bindings deployment_id=1 current_sha=123"
+    setup_template_dir("hello-cloud", subset: %w(bare_replica_set.yml)) do |target_dir|
```

Comment:
Out of curiosity, why the change to `subset:`?

Reply:
hello-cloud has several fixtures that require erb.

Reply:
Cool! Thanks for the clarification, Danny!